protocols/migration/migration_helpers.py

```diff
@@ -33,6 +33,13 @@
 from protocols.reports_5_0_0 import RareDiseaseExitQuestionnaire as RareDiseaseExitQuestionnaire_5_0_0
 from protocols.reports_5_0_0 import CancerExitQuestionnaire as CancerExitQuestionnaire_5_0_0
 
+from protocols.reports_6_0_0 import InterpretationRequestRD as InterpretationRequestRD_6_0_0
+from protocols.reports_6_0_0 import ClinicalReport as ClinicalReport_6_0_0
+from protocols.reports_6_0_0 import InterpretedGenome as InterpretedGenome_6_0_0
+from protocols.reports_6_0_0 import InterpretationRequestRD as InterpretationRequestRD_6_0_0
```

Comment:
this duplicates the import line above and should be importing cancer exit questionnaire, fixed by PR  #305